chains/solana/contracts/programs/ccip-router/src/lib.rs

```diff
@@ -44,6 +44,10 @@ pub mod ccip_router {
 
     use super::*;
 
+    //////////////////////////
+    /// Initialization Flow //
+    //////////////////////////
```

Comment:
Ditto on "section comments" vs modules.

Reply:
Here we can not use modules, as the go bindings do not work 😢

Reply:
Ah well, what can we do!